src/pretty_set.cpp

```diff
@@ -33,13 +33,23 @@ int main(int nargs, char **argv)
 	parse_variables(convert_string(file_storage_location+"variables"));
 	unsigned long t_usec = walltime_usec();
 	
+	pairs = read_rough_pair_data(convert_string(file_storage_location+"processing_files/pairs/"+search_config.filename()+"_rough_pairs_data.csv"));
+
+	uint total_pairs = 0;
+	for(uint i = 0; i<pairs.size(); i++)
+		total_pairs += pairs[i].size();
+
+	if (total_pairs == 0) {
+		cout << "No pairs found" << endl;
+		cout << "Pretty set finished for " << search_config.filename() << ". Runtime: " << 1.0e-6*(walltime_usec() - t_usec)<< " sec" << endl;
+		return 0;
+	}
+
```

Comment:
Should we still output some sort of summary file here? Otherwise will constructor just crash?